config/initializers/dictionary_sign.rb

```diff
@@ -21,6 +21,6 @@
   begin
     DictionarySign.connection.execute("ALTER TABLE words ADD COLUMN word text AS (gloss)")
   rescue ActiveRecord::StatementInvalid => e
-    raise e unless e.message == "SQLite3::SQLException: duplicate column name: word"
+    raise e unless e.message.include?("SQLite3::SQLException: duplicate column name: word")
```

Comment:
:elephant: but perhaps we should be checking that not only is this error in `message` but also its the ONLY error in `message`?